code/plugins/api/users/users/config.php

```
@@ -0,0 +1,234 @@
+	 * @param   number  $cname  The  table
+	 * @param   number  $name   The table
+	 *
+	 * @return  ApiPlugin response object
+	 *
+	 * @since 2.0
+	 */
+	public function getCompParams($cname=null,$name=null)
+	{
+		jimport('joomla.application.component.helper');
+
+		$app   = Factory::getApplication();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$app'. (Unused Code Rules)

(UnusedLocalVariable)